Source/Krypton Components/Krypton.Toolkit/General/OutlookGrid/General/OutlookGridColumn.cs

```diff
@@ -151,7 +151,7 @@ public OutlookGridColumn(DataGridViewColumn? dataGridViewColumn, IOutlookGridGro
         /// <summary>Defines Equals method (interface IEquatable)</summary>
         /// <param name="other">The OutlookGridColumn to compare with</param>
         /// <returns></returns>
-        public bool Equals(OutlookGridColumn other) => DataGridViewColumn!.Name.Equals(other.DataGridViewColumn?.Name);
+        public bool Equals(OutlookGridColumn? other) => DataGridViewColumn!.Name.Equals(other!.DataGridViewColumn!.Name!);
```

Comment:
`other` nullable is ignored, so make it DissallowNull and remove nullable.

Reply:
Was already changed, but forgot to comment here yesterday.